docs/configuration/integrations/github-enterprise.md

```diff
@@ -26,7 +26,7 @@ Follow the instructions below to set up the Codacy Self-hosted integration with
       app:
         name: "codacy" # GitHub App name
         id: "1234" # GitHub App ID
-        privateKey: "" # Contents of the .pem file without newlines
+        privateKey: "" # Contents of the .pem file without newlines or the Begin/End sections
```

Comment:
How about:

```suggestion
        privateKey: "" # Contents of the .pem file without newlines or the BEGIN/END lines
```

I wonder if customers have no problem understanding what they have to do regarding the newlines/line breaks part. :thinking:

Reply:
I'm good with this suggestion. It might not be perfect, but this should at least help :-)